index.bs

```diff
@@ -223,20 +225,89 @@ partial interface XMLHttpRequest {
 };
 </pre>
 
-To <dfn>mark a request for Attribution Reporting eligibility</dfn> given a
-[=header list=] |headers| and an optional {{AttributionReportingRequestOptions}}
-|options|:
+A [=request=] has an associated
+<dfn for=request>Attribution Reporting eligibility</dfn> (an [=eligibility=]).
```

Comment:
Is the upper casing of "Attribution Reporting" correct style? I don't see that in other request members.

Reply:
We use it capitalized throughout the spec, so I'd rather be consistent. IIUC the spec fields are case-insensitive anyway. We can change this later.